src/azul/health.py

```diff
@@ -270,10 +278,11 @@ def elasticsearch(self):
 
     @lru_cache
     def _lambda(self, lambda_name) -> JSON:
+        url = config.lambda_endpoint(lambda_name).set(path='/health/basic',
+                                                      args={'catalog': self.catalog})
+        log.info('Requesting %r', url)
+        response = requests.get(str(url))
```

Comment:
Are you sure `requests.get` doesn't belong in the `try` block? What if a network error occurs before the response is received?

Reply:
Unsure, I was trying to stick to precedent https://github.com/DataBiosphere/azul/blob/develop/src/azul/health.py#L236-L238.